example/00-backup-restore-server-config.yaml

```
@@ -42,4 +42,8 @@ restorationConfig:
   maxFetchers: 6
   embeddedEtcdQuotaBytes: 8589934592
 
-defragmentationSchedule: "0 0 */3 * *"
\ No newline at end of file
+defragmentationSchedule: "0 0 */3 * *"
+
+compressionConfig:
+   enabled: true
+   policy: "zlib"
```

```suggestion
   policy: "gzip"
```
Let't keep `gzip` as the default.

---

actually default is still gzip [here](https://github.com/gardener/etcd-backup-restore/pull/293/files#diff-edf8a5224f35b87bf6e0367f2b78e678d442e8329b0143d0b9653b42be7b752bR29) .... but I can change compressionPolicy in config file as well.

---

Better to be consistent in the examples too, no?